src/server/output.txt

```diff
@@ -0,0 +1,442 @@
+ * Environment: production
```

Comment:
we need to gitignore this file. Also need to change the password.

Reply:
yep this shouldn't be here. deleting and resetting pass